manager/orchestrator/replicatedjob/orchestrator.go

```diff
@@ -47,174 +49,44 @@ func (o *Orchestrator) Run(ctx context.Context) {
 	// method, and so should be the absolute first thing we defer.
 	defer close(o.doneChan)
 
-	// TODO(dperny): this will be a case in the main select loop, but for now
-	// just block until stopChan is closed.
-	<-o.stopChan
-}
-
-// reconcileService reconciles the replicated job service with the given ID by
-// checking to see if new replicas should be created. reconcileService returns
-// an error if there is some case prevent it from correctly reconciling the
-// service.
-func (o *Orchestrator) reconcileService(id string) error {
 	var (
-		service *api.Service
-		tasks   []*api.Task
-		viewErr error
+		services []*api.Service
 	)
-	// first, get the service and all of its tasks
-	o.store.View(func(tx store.ReadTx) {
-		service = store.GetService(tx, id)
 
-		tasks, viewErr = store.FindTasks(tx, store.ByServiceID(id))
+	o.store.View(func(tx store.ReadTx) {
+		// TODO(dperny): figure out what to do about the error return value
+		// from FindServices
+		services, _ = store.FindServices(tx, store.All)
+
+		// there should only ever be 1 cluster object, but for reasons
+		// forgotten by me, it needs to be retrieved in a rather roundabout way
+		// from the store
+		// TODO(dperny): figure out what to do with this error too
+		clusters, _ := store.FindClusters(tx, store.All)
+		if len(clusters) == 1 {
```

Comment:
And `TODO: figure out what to do if there's more than 1 cluster` ;) ?

Might it make sense to have a `store.FindCluster` that returns the One and panics if there's an error or a different # ?

Reply:
There shouldn't ever be more than one cluster. AFAIK it's impossible, and there's no plan in the pipeline to ever add anything like more than one cluster.

Reply:
Can't hurt to panic in one place though, if that ever changes?

Reply:
Will add it in my next PR, coming shortly.